modules/default-flatpaks/default-flatpaks.sh

```diff
@@ -103,23 +103,98 @@ configure_lists () {
 }
 
 echo "Enabling flatpaks module"
-mkdir -p /usr/etc/flatpak/{system,user}
+mkdir -p /usr/share/bluebuild/default-flatpaks/{system,user}
 systemctl enable -f system-flatpak-setup.service
 systemctl enable -f --global user-flatpak-setup.service
 
-# Check that `system` is present before configuring
+# Check that `system` is present before configuring. Also document list files with additional information.
 if [[ ! $(echo "$1" | yq -I=0 ".system") == "null" ]]; then
     configure_flatpak_repo "$1" "system"
+    system_install_list_doc="/usr/share/bluebuild/default-flatpaks/system/install"
+    system_remove_list_doc="/usr/share/bluebuild/default-flatpaks/system/remove"
+    echo -e "# This file utilizes maintainer's configuration for \`system flatpaks install\` used by \`default-flatpaks\` BlueBuild module.
```

Comment:
These docs echoes should be moved to the correct files with the correct file structure directly in the module source. Just create a new directory called like `config` or smn with the correct file structure and docs inside files, then just copy it into place in the script.

Reply:
Would you do that before I make my changes?

Reply:
@xynydev Done